cfgov/cfgov/urls.py

```diff
@@ -289,6 +300,16 @@
     url(r'^token-provider/', token_provider),
 ]
 
+with flagged_urls('CCDB5_RELEASE') as _url:
+    apiBase = '^data-research/consumer-complaints/api/v1'
+    ccdb5_patterns = [
+        _url(apiBase + '/_suggest', CCDB5.suggest),
+        _url(apiBase + '/(?P<id>[0-9]+)$', CCDB5.document),
+        _url(apiBase, CCDB5.search)
+    ]
+urlpatterns += ccdb5_patterns
```

Comment:
Instead of each of these being its own `flagged_url('CCDB5_RELEASE`, …)` line, this way we can flag a bunch of URLs in the same context. It's an attempt to make multiple URLs flagged with the same flag a little easier to read.